sgeop/nodes.py

```diff
@@ -44,14 +60,26 @@ def split(split_points, cleaned_roads, crs, eps=1e-4):
                 )
                 gdf_split["_status"] = "changed"
                 cleaned_roads = pd.concat(
-                    [
-                        cleaned_roads.drop(to_be_dropped),
-                        gdf_split,
-                    ],
+                    [cleaned_roads.drop(to_be_dropped), gdf_split],
                     ignore_index=True,
                 )
 
-    return cleaned_roads.reset_index(drop=True)
+    cleaned_roads = (
+        cleaned_roads.reset_index(drop=True)
+        .drop(columns=0, errors="ignore")
+        .set_geometry("geometry")
+        .set_crs(crs)
```

Comment:
What is the reason for these?

Reply:
Relic from testing experiments. Will remove.